Sources/SwifterSwift/Foundation/MeasurementExtensions.swift

```diff
@@ -49,4 +49,46 @@ public extension Measurement where UnitType == UnitAngle {
     }
 }
 
+public extension Measurement {
+    /// SwifterSwift: Adds two values and stores the result in the left-hand-side variable.
+    /// - Parameters:
+    ///   - lhs: The first value to add.
+    ///   - rhs: The second value to add.
+    static func += (lhs: inout Measurement<UnitType>, rhs: Measurement<UnitType>) {
```

Comment:
Isn't `<UnitType>` assumed? If so, remove for all functions.
```suggestion
    static func += (lhs: inout Measurement, rhs: Measurement) {
```